js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,53 @@
+function isNumber(...args) {
+  let result = true;
+
+  for (const arg of args) {
+    if (typeof arg !== 'number') {
+      result = false;
+      break;
+    }
+  }
+
+  return result;
+}
+
+function isPositiveInteger(...args) {
+  let result = true;
```

Comment:
Not required when you use array functions.

Reply:
correct, no longer exist in latest code pushed